Cargo.toml

```diff
@@ -19,14 +19,14 @@ doctest = false
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-prodash = { version = "19.0.1", features = ["render-line", "render-line-crossterm", "signal-hook", "render-line-autoconfigure", "progress-tree", "unit-bytes"] }
+prodash = { version = "20.0.0", features = ["render-line", "render-line-crossterm", "signal-hook", "render-line-autoconfigure", "progress-tree", "unit-bytes"] }
 memmap2 = "0.5.7"
 num_cpus = "1.13.1"
 flume = { version = "0.10.14", default-features = false }
 open = "3.0.2"
 clap = { version = "3.2.20", features = ["derive"] }
 bytesize = "1.1.0"
-bstr = "0.2.17"
+bstr = "1.0.0"
```

Comment:
Is updating `bstr` needed for updating prodash? I'm actually not finding what we're using bstr for. I see `use bstr::ByteSlice;` in `render.rs` and `visualize.rs` but I'm not sure what it's doing.